integration/common/openlineage/common/provider/dbt/utils.py

```diff
@@ -102,11 +104,35 @@ def get_parent_run_metadata():
     The parent job that started the dbt command. Usually the scheduler (Airflow, ...etc)
     """
     parent_id = os.getenv("OPENLINEAGE_PARENT_ID")
+    root_parent_id = os.getenv("OPENLINEAGE_ROOT_PARENT_ID")
```

Comment:
Not related to current PR, but this implies that job namespace or name cannot contain slashes, otherwise parent_id cannot be parsed. Am I right?

Reply:
@dolfinus as by convention for `OPENLINEAGE_PARENT_ID`. 

To fix that, for example we could pass the parent context via json.